### PR TITLE
SSH Keys: client

### PR DESCRIPTION
**Pull request by ksol**
I tried to keep the conventions in place, let me know if some naming is not correct

### CODE REVIEW

File: src/Keys/index.js [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+  all() {
+    return unpackData(this._client.apiClient().get('/keys'), 'keys')
+  }
+
+  /**
+   * Show one specific key
+   * @return {Promise<Key | APIError>} A key
+   */
+  show(id) {
+    return unpackData(this._client.apiClient().get(`/keys/${id}`), 'key')
```
Comment (EtienneM) [inline src/Keys/index.js]: It seems that both `show` and `create` endpoints return something like:

```json
{
  "key" : [
    {
...
    }
  ]
}
```

https://developers.scalingo.com/keys#get-a-precise-ssh-key

This is odd but it is what it is :sweat_smile: 

Can you confirm the dev doc is correct? If it is, Don't you want to return just a key (and not an array containing a single key)?

File: src/Keys/index.js [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+}
+
+/**
+ * @typedef {Object} Key
+ * @see https://developers.scalingo.com/scm_integrations#createlink-an-scm-integration-with-your-account
+ * @property {String} id Unique key ID
+ * @property {String} name Key name
+ * @property {String} content
+ * @property {String} fingerprint Fingerprint of the SSH key
+ * @property {Date} created_at Creation date of the SCM integration
```
Comment (EtienneM) [inline src/Keys/index.js]:
```suggestion
 * @property {Date} created_at Creation date of the SSH key
```

File: src/Keys/index.js [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+   * @return {Promise<? | APIError>}
+   */
+  destroy(id) {
+    return unpackData(this._client.apiClient().delete(`/keys/${id}`))
+  }
+}
+
+/**
+ * @typedef {Object} Key
+ * @see https://developers.scalingo.com/scm_integrations#createlink-an-scm-integration-with-your-account
```
Comment (EtienneM) [inline src/Keys/index.js]:
```suggestion
 * @see https://developers.scalingo.com/keys
```

File: src/Keys/index.js (hunk 1 above)
Comment (ksol) [reply src/Keys/index.js]: I think the doc is wrong, I'm not seeing arrays returned. I'll open an issue

Comment (ksol) [reply src/Keys/index.js]: https://github.com/Scalingo/developers-documentation/issues/71

**Pull request merged**